javascript/javascript3/week2/promiseVisual/main.js

```diff
@@ -0,0 +1,59 @@
+
+
+//To translate one by one
+
+/*
+const translateOneByOne=()=>{
+
+    //promise for red circle
+    moveElement(document.querySelector('li:nth-child(1)'), {x: 20, y: 300})
+        .then(() => {
+            console.log('Element has been moved');
+
+            //promise for blue circle
+            return moveElement(document.querySelector('li:nth-child(2)'), {x: 400, y: 300})
+        })
+
+        
+        .then(() => {
+            console.log('Element has been moved');
+
+            //promise for green circle
+            return  moveElement(document.querySelector('li:nth-child(3)'), {x: 400, y: 20})
+        })
+    
+        .then(() => {
+            console.log('Element has been moved');
+            return moveElement(document.querySelector('span'), {x: 200, y: 150})
+        });
+         
+}
+translateOneByOne();*/
```

Comment:
Correct, but very hard to read. Also, try to avoid pushing commented out code to Githib.